drivers/input/touchscreen/ef1e_tp/Makefile

```diff
@@ -0,0 +1,18 @@
+ifneq ($(KERNELRELEASE),)
+obj-$(CONFIG_TOUCHSCREEN_EF1E_TP) += ef1e_tp.o
+ef1e_tp-objs += tp_input.o \
+	tp_driver.o
```

Comment:
tp name is too generic, can we change the prefix tp to ef1e?

Reply:
Sure.